src/__stories__/chip-story.tsx

```diff
@@ -144,19 +157,53 @@ export const MultipleSelection: StoryComponent<Omit<Args, 'closable'>> = ({inver
         </ChipBackgroundContainer>
     );
 };
+export const NavigableChip: StoryComponent<Omit<Args, 'closable'>> = ({
+    inverse,
+    icon,
+    badge,
+    active: chipActive,
+    href: hrefProp,
```

Comment:
I won't make href configurable in this story, the whole point of the story is to show the touchable+active use case, so it doesn't make sense to allow configuring href to undefined here